templates/docs/examples/layouts/empty-state/user-triggered.html

```diff
@@ -0,0 +1,47 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Empty State / User action triggered  {% endblock %}
+
+{% block standalone_css %}layouts{% endblock %}
+
+{% block content %}
+<div class="p-strip is-shallow u-fixed-width">
```

Comment:
We should use sections instead of divs as they are more semantically correct. We also don't add `p-strip` classes to divs, they go on each section

Reply:
If we put  search box inside a single section then it will shift up. 

![image](https://user-images.githubusercontent.com/24393395/151743383-398b97e9-618e-4acb-b1f8-36bd1ec05d05.png)

After the section is added:
![image](https://user-images.githubusercontent.com/24393395/151743487-2cd7e8d6-f4c4-4326-85bb-7f42905415b6.png)